app/Controllers/Course.hs

```diff
@@ -43,4 +53,4 @@ depts = do
         return $ sort . nub $ map g coursesList :: SqlPersistM [String]
     return $ createJSONResponse deptList
     where
-        g = take 3 . T.unpack . coursesCode . entityVal
+        g = take 3 . T.unpack . coursesCode . entityVal
```

Comment:
This symbol on GitHub means you have not ended the file with a new line. You should make sure all files you added/changed end with a new line, and configure your text editor to ensure that all files end with a new line, to make sure this doesn't happen.